modules/op-configuring-custom-sa-and-scc.adoc

```diff
@@ -15,21 +15,21 @@ The default `pipeline` SA allows using a user id outside of the namespace range.
 .Example: Custom SA and SCC for used id `1000`
```

Comment:
Consider separating out the API object examples at a later date.

Reply:
https://issues.redhat.com/browse/RHDEVDOCS-5506 created